src/test/java/com/google/api/codegen/DiscoveryGeneratorTestBase.java

```diff
@@ -53,6 +58,22 @@ public DiscoveryGeneratorTestBase(
     this.name = name;
     this.discoveryDocFileName = discoveryDocFileName;
     this.gapicConfigFileNames = gapicConfigFileNames;
+
```

Comment:
Does this file automatically generate the snippets from the checked-in discoveries and compare the output to the baselines?